### PR TITLE
refactor: undry some things for more conciseness

### PR DESCRIPTION
**Pull request by blaggacao**

### CODE REVIEW

File: run/action.yaml [hunk 1]
```diff
@@ -105,18 +92,23 @@ runs:
...
+      env:
+        GITHUB_PAT: ${{ inputs.github_pat }}
+        NIX_KEY_PATH: ${{inputs.key_path}}
       with:
         extra_nix_config: |
           accept-flake-config = true
           trusted-users = runner
-          access-tokens = github.com=${{ inputs.github_pat }}
-          secret-key-files = ${{ env.NIX_KEY_PATH }}
+          access-tokens = github.com="$GITHUB_PAT"
```
Comment (nrdxp) [inline run/action.yaml]: this isn't gonna work, its not a shell snippet

**Review by nrdxp — COMMENTED** [review]
a little scary without any tests, but the impetus is good

File: run/action.yaml (hunk 1 above)
Comment (blaggacao) [reply run/action.yaml]: Yeah, just realized this a little later.  It's fixed now. Thx! :smile:

**Comment by blaggacao** [comment]
I guess somebody will have to suffer, if this breaks something.

I'm considerate in highly prioritizing technical debt at this early stage (see warning in the readme).

**Comment by nrdxp** [comment]
yeah, but we need some tests before we merge too many untested changesets for the same reason :sweat_smile:

**Pull request merged**